ext/AvizExt/viz/spatial.jl

```diff
@@ -215,6 +216,127 @@ function ADRIA.viz.map!(
     )
 end
 
+"""
+    ADRIA.viz.connectivity(dom::Domain, network::SimpleWeightedDiGraph, conn_weights::AbstractVector{<:Real}; opts::Dict=Dict(), fig_opts::Dict=Dict(), axis_opts::Dict=Dict()) 
+    ADRIA.viz.connectivity!(g::Union{GridLayout, GridPosition}, dom::Domain,  network::SimpleWeightedDiGraph, conn_weights::AbstractVector{<:Real}; opts::Dict=Dict(), axis_opts::Dict=Dict()) 
+
+Produce visualisation of connectivity between reef sites with node size and edge visability
+weighted by the connectivity values and node weights.
+
+# Example
+```julia
+dom = ADRIA.load_domain("<Path to Domain>")
+
+in_conn, out_conn, network = ADRIA.connectivity_strength(dom; out_method=eigenvector_centrality)
+
+ADRIA.viz.connectivity(
+    dom,
+    network,
+    out_conn;
+    opts=opts,
+    fig_opts=fig_opts,
+    axis_opts=axis_opts
+)
+```
+# Arguments
+- `dom` : Domain
+- `network` : SimpleWeightedDiGraph calculated from the connectivity matrix
+- `conn_weights` : Connectivity weighted for each node
+- `opts` : AvizOpts
+    - `edge_color`, vector of colours for edges. Defaults to reasonable weighting
+    - `node_color`, vector of colours for node. Defaults to conn_weights
```

Comment:
Reference to arguments/variables should use backticks

```suggestion
    - `node_color`, vector of colours for node. Defaults to `conn_weights`.
```